docs/platforms/javascript/common/configuration/integrations/http.mdx

```diff
@@ -45,12 +45,22 @@ If set to false, no breadcrumbs will be captured.
 
 ### `ignoreOutgoingRequests`
 
-_Type: `(url: string) => boolean`_
+_Type: `(url: string, request: RequestOptions) => boolean`_
 
-Allows you to define a method to filter out outgoing requests based on the URL. If the method returns `true`, the request will be ignored.
+Allows you to define a method to filter out outgoing requests based on the URL. If the method returns `true`, no spans or breadcrumbs will be captured for the outgoing request.
+
+The callback function receives two arguments:
+
+- `url`: The full URL of the outgoing request, including the protocol, host, port, path and query string. For example: `https://someService.com/users?name=John`.
```

Comment:
l: Ultra nit, but I prefer to use `example.com` for any example urls. Never know what kind of website will live under `someService.com` one day and what user might try open it, even if it's just in docs.

```suggestion
- `url`: The full URL of the outgoing request, including the protocol, host, port, path and query string. For example: `https://example.com/users?name=John`.
```

Reply:
Good point! 😅